red/api/theme.js

```diff
@@ -128,6 +128,15 @@ module.exports = {
             }
         }
 
+        if (theme.customScripts) {
+            var urls = []
+            for (var j = 0; j < theme.customScripts.length; j++) {
```

Comment:
The user should be allowed to specify either a single string or an array of strings.